db/migrate/20200701215842_create_orders.rb

```diff
@@ -0,0 +1,12 @@
+class CreateOrders < ActiveRecord::Migration[6.0]
+  def change
+    create_table :orders do |t|
+      t.string :table_number
+      t.decimal :prices
+      t.string :payment_method
```

Comment:
payment_method có các loại method nào ?

Reply:
Dạ về phương thức thanh toán bọn e sẽ có là thanh toán tiền mặt tại quầy và người dùng quẹt thẻ.
Nó có 2 trường hợp nên em sẽ chuyển nó về enum